lib/config/presets/internal/replacements.ts

```diff
@@ -3,7 +3,21 @@ import type { Preset } from '../types';
 export const presets: Record<string, Preset> = {
```

Comment:
The PR title could probably be changed to `feat(config): add replacement preset for flake8-pathlib`, so that it has the config scope?